text/0000-Project-Goals-2025h2.md

```diff
@@ -0,0 +1,754 @@
+[wg-llvm]: https://forge.rust-lang.org/compiler/working-areas.html
+[wg-macros]: https://github.com/rust-lang/wg-macros
+[wg-mir-opt]: https://forge.rust-lang.org/compiler/working-areas.html
+[wg-parallel-rustc]: https://forge.rust-lang.org/compiler/working-areas.html
+[wg-polonius]: https://forge.rust-lang.org/compiler/working-areas.html
+[wg-rustc-dev-guide]: https://forge.rust-lang.org/compiler/working-areas.html
+[wg-safe-transmute]: https://github.com/rust-lang/project-safe-transmute
+[wg-secure-code]: https://github.com/rust-secure-code/wg
+[wg-security-response]: https://github.com/rust-lang/wg-security-response
+[wg-triage]: https://www.rust-lang.org/governance/teams
+[windows]: https://www.rust-lang.org/governance/teams
+
+
+[Bastian Kersting]: https://github.com/1c3t3a
+[Amanieu d'Antras]: https://github.com/Amanieu
+[@BN]: https://github.com/BN
+[Benno Lossin]: https://github.com/BennoLossin
+[Boxy]: https://github.com/BoxyUwU
+[Alice Ryhl]: https://github.com/Darksonn
+[Guillaume Gomez]: https://github.com/GuillaumeGomez
+[James]: https://github.com/Jamesbarford
+[Pete LeVasseur]: https://github.com/PLeVasseur
+[Ralf Jeffery]: https://github.com/RalfJ
+[Sparrow Li]: https://github.com/SparrowLii
+[Wesley Wiser]: https://github.com/WesleyWiser
+[Manuel Drehwald]: https://github.com/ZuseZ4
+[Aapo Alasuutari]: https://github.com/aapoalas
+[Alona Enraght-Moony]: https://github.com/adotinthevoid
+[Jon Bauman]: https://github.com/baumanj
+[@boxy]: https://github.com/boxy
+[Carol Nichols]: https://github.com/carols10cents
+[Taylor Cramer]: https://github.com/cramertj
+[David Wood]: https://github.com/davidtwco
+[Ding Xiang Fei]: https://github.com/dingxiangfei2009
+[David Tolnay]: https://github.com/dtolnay
+[Eric Huss]: https://github.com/ehuss
+[Ed Page]: https://github.com/epage
+[Folkert de Vries]: https://github.com/folkertdev
+[Frank King]: https://github.com/frank-king
+[Ian McCormack]: https://github.com/icmccorm
+[Tugce Ciftci]: https://github.com/imperio
+[Jack Huey]: https://github.com/jackh726
+[@jackwrenn]: https://github.com/jackwrenn
+[Jakob Koschel]: https://github.com/jakos-sec
+[Josh Triplett]: https://github.com/joshtriplett
+[Jack Wrenn]: https://github.com/jswrenn
+[Jakub Beránek]: https://github.com/kobzol
+[lcnr]: https://github.com/lcnr
+[Rémy Rakic]: https://github.com/lqd
+[Marco Ieni]: https://github.com/marcoieni
+[Niko Matsakis]: https://github.com/nikomatsakis
+[Predrag Gruevski]: https://github.com/obi1kenobi
+[Oliver Scherer]: https://github.com/oli-obk
+[Vadim Petrochenkov]: https://github.com/petrochenkov
+[@ranger-ross]: https://github.com/ranger-ross
+[Ben Kimock]: https://github.com/saethlin
+[Scott McMurray]: https://github.com/scottmcm
+[Santiago Pastorino]: https://github.com/spastorino
+[Tyler Mandry]: https://github.com/tmandry
+[Tomas Sedovic]: https://github.com/tomassedovic
+[TC]: https://github.com/traviscross
+[Weihang Lo]: https://github.com/weihanglo
+[@whlo]: https://github.com/whlo
```

Comment:
Some of my handle are wrong.

`@whlo` should be either `Weihang Lo` or `@weihanglo`.

https://github.com/rust-lang/rust-project-goals/pull/375

Reply:
D'oh! Thanks.